src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -432,6 +434,8 @@ public void commit(int type) {
             commitModuleList();
         } else if (type == 2) {
             commitContactList();
+        } else if (type == 4) {
+            commitEventList();
```

Comment:
If possible, perhaps we can consider using enum to avoid the use of magic number?

Reply:
Yes I was coming around to doing that but forgot. Thanks for the reminder.